redash/utils/__init__.py

```diff
@@ -155,6 +155,6 @@ def base_url(org):
     if settings.MULTI_ORG:
         return "https://{}/{}".format(settings.HOST, org.slug)
 
-    return settings.HOST
+    return "https://{}".format(settings.HOST)
```

Comment:
Unfortunately not everyone is using `https`... I think uniforming should be in the opposite direction, and make the MULTI_ORG branch of the if not use https by default.